### ISSUE TITLE
Expose Movies Path to Blueprints to change change video played at Runtime

### ISSUE THREAD

**Issue by sbnewsom** [ue4plugins/LoadingScreen#19]
How would I go about exposing the classes to blueprint so we can change MoviePaths to DefaultScreen struct at Runtime? I'm trying to have it so through blueprints that it can change the moviepaths, so when I make a change to level, a cutscene will play out instead of a static array we set in the Project Settings.

**Comment by HoaxXx** [comment]
Yes, it would be nice to be able to change things like that or the array with "tips" in runtime.

**Comment by sbnewsom** [comment]
Actually, since I posted this, I have made the Loading system accessible through blueprints with the ability to change any settings at any moment at runtime. I might clean up the code and push it to the repository!

**Comment by HoaxXx** [comment]
That would be very welcome.

**Comment by OfficialNoms** [comment]
Have you managed to do this? The ability to control some aspects of this plugin would be very useful.

**Comment by BorderlineInappropriateGameStudios** [comment]
Hey guys, created a struct variable and can see some settings in my blueprint but it doesn't seem to update at all, always sticks to the default settings. How have you got this working?
